scripts/cc-cli/src/utils/bond.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiPromise } from "creditcoin-js";
+import { initKeyringPair } from "./account";
+
+export async function bond(
+  stashSeed: string,
+  controllerAddress: string,
+  amount: number,
+  rewardDestination: "Staked" | "Stash" | "Controller",
+  api: ApiPromise
+) {
+  const bondTx = api.tx.staking.bond(
+    controllerAddress,
+    `${amount}000000000000000000`, // Add 18 zeros to convert to micro units TODO: Improve amount handling
+    rewardDestination
+  );
+
+  const stashKeyring = initKeyringPair(stashSeed);
+
+  const hash = await bondTx.signAndSend(stashKeyring);
+
+  return hash.toHex();
+}
+
+export function parseRewardDestination(rewardDestination: string) {
+  if (rewardDestination === "staked") {
+    return "Staked";
+  } else if (rewardDestination === "stash") {
+    return "Stash";
+  } else if (rewardDestination === "controller") {
+    return "Controller";
+  } else {
+    console.log(
+      "Invalid reward destination, must be one of 'staked', 'stash', or 'controller'"
+    );
+    process.exit(1);
```

Comment:
Hmm, that's an interesting way of error handling. I was expecting to throw an exception. 

What do others think ?

Reply:
Been reviewing error handling and its quite messy. Several commands and functions do it differently.

I would refactor error handling to make it consistent, making all `utils` throw errors, while `commands` do the handling (exit, print errors and instructions).